agent.py

```diff
@@ -45,6 +50,7 @@ def __init__(self, env, policy, vision=Vision(), callbacks=[]):
         interface.
             vision (Vision): Processes raw environment output
```

Comment:
Tutaj jest złamana linia szybciej niż dla kolejnego parametru. Trzeba określić po ilu znakach łamiemy linię i się trzymać (100 to chyba jest standard w tym momencie, tak ja mam ustawione i tak kiedyś znalazłem w necie).